src/Go/boardAnalysis/goAI.ts

```diff
@@ -57,6 +56,11 @@ export function makeAIMove(boardState: BoardState, useOfflineCycles = true): Pro
           return play;
         }
 
+        if (currentMoveCount !== Go.currentGame.previousBoards.length) {
+          console.error("AI move attempted, but the board state has changed.");
```

Comment:
Let's downgrade this to a warn, it's something to know about but doesn't indicate an error in anyone's (including the game's) code.